spec/std/process_spec.cr

```diff
@@ -215,7 +215,7 @@ describe Process do
   end
 
   describe "find_executable" do
-    pwd = Process::INITIAL_PWD
+    pwd = Dir.current
```

Comment:
I'm not sure: by the time the spec is run, the current directory may have been changed.